lib/active_encode/engine_adapters/ffmpeg_adapter.rb

```diff
@@ -34,7 +34,13 @@ def create(input_url, options = {})
         FileUtils.mkdir_p working_path("outputs", new_encode.id)
 
         # Extract technical metadata from input file
-        `#{MEDIAINFO_PATH} --Output=XML --LogFile=#{working_path("input_metadata", new_encode.id)} "#{input_url}"`
+        curl_option = if options && options[:headers]
+                        headers = options[:headers].map { |k, v| "#{k}: #{v}" }
+                        (["--File_curl=HttpHeader"] + headers).join(",")
```

Comment:
It really joins with a comma between `--File_curl` and the actual headers?  That seems odd.

Reply:
Yes, it's pretty odd.. but it worked